### PR TITLE
chore(llm-tool): add repetition_penalty example

### PR DESCRIPTION
**Pull request by mzbac**
Add the example of the repetition penalty and also update the deprecated indexing for `logits[-1, axis: 1]`.

### CODE REVIEW

**Review by davidkoski — APPROVED** [review]
Looks great, thank you for another contribution!
